src/imgtools/modalities/__init__.py

```
@@ -0,0 +1,5 @@
+from .dose import Dose
```

Ruff (0.8.2)

`.dose.Dose` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)